mesonbuild/modules/qt5.py

```diff
@@ -133,30 +133,30 @@ def preprocess(self, state, args, kwargs):
             else:
                 basename = os.path.split(rcc_files[0])[1]
                 name = 'qt5-' + basename.replace('.', '_')
-            rcc_kwargs = {'input' : rcc_files,
-                    'output' : name + '.cpp',
-                    'command' : [self.rcc, '-o', '@OUTPUT@', '@INPUT@'],
-                    'depend_files' : qrc_deps,
-                    }
+            rcc_kwargs = {'input': rcc_files,
+                          'output': name + '.cpp',
+                          'command': [self.rcc, '-o', '@OUTPUT@', '@INPUT@'],
+                          'depend_files': qrc_deps,
+            }
```

Comment:
__[flake8]__
* [E124] closing bracket does not match visual indentation

<!-- Comment by [SideCI](https://sideci.com) -->